owlplug-client/src/main/resources/owlplug.css

```
@@ -158,54 +158,67 @@ Text {
-.chip-label {
-    -fx-font-size: 12.0;
-    -fx-background-color: #E0E0E0;
-    -fx-background-radius: 40px;
-    -fx-padding: 2px 8px 2px 8px;
-    -fx-pref-height: 32px;
-    -fx-alignment: center-left;
-    -fx-spacing: 8;
-    -fx-text-fill: black;
-}
-
 .chip-view .chip Label {
```

Stylelint (17.12.0)

[error] Expected "Label" to be "label" (selector-type-case)

(selector-type-case)